hello-world/enclave/src/test/java/com/r3/conclave/sample/enclave/MockTest.java

```diff
@@ -0,0 +1,31 @@
+package com.r3.conclave.sample.enclave;
+
+import com.r3.conclave.host.EnclaveHost;
+import com.r3.conclave.host.EnclaveLoadException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Tests the enclave fully in-memory in a mock environment without the web host.
+ */
+public class MockTest {
+    @Test
```

Comment:
Question: do you think we really need a MockTest for these features? I.e. the persistentMap and persistentFileSystem look very related to simulation/debug/release modes, not sure it makes sense even to talk about Mock mode, imho